server/app/controllers/checks_controller.py

```diff
@@ -208,6 +225,7 @@ def disable_check(parameters):
                 db.session.commit()
                 ChecksController.remove_scheduled_job(check_id)
 
+    # This method returns the status of the check.
```

Comment:
This just begs the question; what does status mean?  What are its possible values?

Reply:
This method returns `true` if the check is active and returns `false` otherwise.

Reply:
I'll rephrase the comment to include the return types.